helm/hpcc/templates/eclagent.yaml

```diff
@@ -50,6 +50,13 @@ data:
 {{- include "hpcc.addResources" (dict "me" .me.resources) | indent 12 }}
 {{- $appCmd := printf "%s %s %s _HPCC_ARGS_" $apptype (include "hpcc.configArg" .me) (include "hpcc.daliArg" .root) }}
 {{ include "hpcc.addCommandAndLifecycle" (. | merge (dict "command" $appCmd)) | indent 12 }}
+{{- if index .me "env" }}
+            env:
+{{- range .me.env }}
```

Comment:
can you do {{ toYaml .me.env | indent 2 }} instead of range?

Reply:
I could on this one - I can't on any where we need to add the SENTINEL entry (though I suppose I could do that by adding to the env dictionary). Eventually I expect the resulting env may well be a merge of a global and a local one.